common/src/main/java/muramasa/gregtech/data/Materials.java

```diff
@@ -686,23 +680,23 @@ public class Materials {
-    //public static Material HydroCrackedButadiene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_butadiene", 0xe86900, NONE));
+    public static Material HydroCrackedButene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_butene", 0xcf5005, NONE));
+    public static Material HydroCrackedButadiene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_butadiene", 0xe86900, NONE));
     public static Material HydroCrackedHeavyFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_heavy_fuel", 0xffff00, NONE));
     public static Material HydroCrackedLightFuel = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_light_fuel", 0xffff00, NONE));
     public static Material HydroCrackedNaphtha = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_naphtha", 0xffff00, NONE));
     public static Material HydroCrackedRefineryGas = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "hydro_cracked_refinery_gas", 0xffffff, NONE));
     public static Material SteamCrackedEthane = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_ethane", 0xc8c8ff, NONE));
-    //public static Material SteamCrackedEthylene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_ethylene", 0xe1e1e1, NONE));
+    public static Material SteamCrackedEthylene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_ethylene", 0xe1e1e1, NONE));
     public static Material SteamCrackedPropane = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_propane", 0xfae250, NONE));
-    //public static Material SteamCrackedPropene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_propene", 0xffdd55, NONE));
+    public static Material SteamCrackedPropene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_propene", 0xffdd55, NONE));
     public static Material SteamCrackedButane = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_butane", 0xb6371e, NONE));
-    //public static Material SteamCrackedButene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_butene", 0xcf5005, NONE));
-    //public static Material SteamCrackedButadiene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_butadiene", 0xe86900, NONE));
+    public static Material SteamCrackedButene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_butene", 0xcf5005, NONE));
+    public static Material SteamCrackedButadiene = AntimatterAPI.register(Material.class, new Material(GTIRef.ID, "steam_cracked_butadiene", 0xe86900, NONE));
```

Comment:
merge uncommented them, need to be recommented

Reply:
Fixed